rqt_image_view/src/rqt_image_view/image_view.cpp

```diff
@@ -77,11 +77,20 @@ void ImageView::initPlugin(qt_gui_cpp::PluginContext& context)
 
   ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("image"));
   connect(ui_.save_as_image_push_button, SIGNAL(pressed()), this, SLOT(saveImage()));
+
+  pub_topic_custom_ = false;
```

Comment:
I've left the boolean variable as I find that behaviour useful and the implementation is good enough imo.